src/VisualStudioExtension/src/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Oops, Visual Studio "helped" by making a copy of this file and I didn't notice on initial commit